### DISCUSSION TITLE
VERCEL DEPLOYEMENT

### DISCUSSION THREAD

**Discussion by validateITNikita** [nuxt/nuxt#18487] [Questions]
Hello!

I am deploying my nuxt 3 app on vercel.

i have static folder which contain many json's. I am trying to access them with api.

this is api file ./server/api/metadata/[id].ts
```
export default defineEventHandler((event) => {
	
	let id : string = event.context.params.id;

	let assets = `/assets/metadata/${id}`;
	
	let filePath = path.join(process.cwd(), assets);
	let data = fs.readFileSync(filePath);
	
	return data;
})
```

But I am getting the error on production website

VERCEL: "message": "ENOENT: no such file or directory, open '/var/task/assets/metadata/55.json'"

Netlify: displaying file correctly

Maybe i need to set something or configure to use it with vercel. 
But i cant find anything.
If you need additional information please ask in the comment.
Thank you for help!

**Comment by cosbgn** [comment]
@validateITNikita did you find a solution?

**Reply by validateITNikita** [reply]
i ma sorry for late response, but yes i found it

try this. Put your assets files in /server/assets/ folder and after that use this to get it 
```

export default defineEventHandler(async (event) => {
    if (!event.context.params)
        return {
            data: "ERROR! Try again!",
        };

    let id: string = event.context.params.id;
    let assetsPath = **your path with required file id**;
    let data = await useStorage().getItem(assetsPath);
    return data;

});

```

**Reply by felixranesberger** [reply]
I just tried adding a example file to the `server/assets` path and it is publicly available.
